autofill/src/driver.py

```diff
@@ -32,6 +33,7 @@ class AutofillDriver:
     )  # delay initialisation until XML is selected and parsed
     driver_callable: Callable[[bool], WebDriver] = attr.ib(default=get_chrome_driver)
     headless: bool = attr.ib(default=False)
+    export_pdf: bool = attr.ib(default=False)
```

Comment:
running the tool in pdf creation mode should be distinct from running the tool normally, so i would like to avoid modifying this file if possible. a better structure would be instantiating `PdfExporter` directly from `autofill.py` like this:

```py
def main(skipsetup: bool, browser: str, exportpdf: bool) -> None:
    try:
        if exportpdf:
            PdfExporter().execute()
        else:
            AutofillDriver(driver_callable=browsers[browser], export_pdf=exportpdf).execute(skipsetup)
    except Exception as e:
        print(f"An uncaught exception occurred: {TEXT_BOLD}{e}{TEXT_END}")
        input("Press Enter to exit.")
```

then, `PdfExporter` would be responsible for instantiating a `CardOrder` from the local directory + configuring the threads and progress bars for downloading images. it would also be ideal if we didn't have to rely on the user pressing enter after seeing the download bar complete. what are your thoughts?

lmk if you don't have time to update the code in this way and i can do it at a later point :)